os/fs/smartfs/smartfs_utils.c

```diff
@@ -870,7 +870,7 @@ int smartfs_mount(struct smartfs_mountpt_s *fs, bool writeable)
 		fs->fs_workbuffer = (char *)kmm_malloc(256);
 	}
 
-	/* Now add ourselves to the linked list of SMART mounts */
+	/* Now add ourselves to the linked list of jSmartFs mounts */
```

Comment:
This one I feel should be kept as the intended reference is not to a SMART formatted block but rather to a Filesystem entity,